### PR TITLE
add -r parameter for create and init command

### PR DESCRIPTION
**Pull request by AndyWu2015**

### CODE REVIEW

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java [hunk 1]
```diff
@@ -55,6 +55,8 @@
 
 	public static final String TEMPLATES_VERSION = "1+";
 
+	private static File zipFile = null;
```
Comment (gamerson) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java]: We shouldn't need a static zip file here.

Comment (gamerson) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java]: Its ok if that is calculated everytime, or are you seeing this get calculated multiple times in a single call?

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/liferay/liferay-blade-cli/pull/37. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java (hunk 1 above)
Comment (gamerson) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java]: I see it shouldn't hurt to cache it.

Comment (AndyWu2015) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java]: yeah , there are multiple calculation in a single call

**Pull request closed without merging**